lib/common/tests/acl/Makefile.am

```diff
@@ -1,19 +1,26 @@
 #
-# Copyright 2021 the Pacemaker project contributors
+# Copyright 2022 the Pacemaker project contributors
```

Comment:
Copyrights are updated like 2021-2022

Copyrights indicate the year the code was published, so the notice covers the entire lifetime of all changes